src/shared/search.ts

```diff
@@ -19,6 +19,15 @@ function getDefaultGlobOptions(): glob.GlobOptions {
   }
 }
 
+export function getGlobOptions(
+  followSymbolicLinks?: boolean
+): glob.GlobOptions {
+  return {
+    followSymbolicLinks,
+    ...getDefaultGlobOptions()
```

Comment:
I think the order is wrong here. `followSymbolicLinks: true` from `getDefaultGlobOptions` will always override the `followSymbolicLinks` property, won't it?

Reply:
You are correct